ui/pages/confirmations/confirm-transaction/confirm-token-transaction-switch.js

```diff
@@ -148,6 +148,14 @@ export default function ConfirmTokenTransactionSwitch({ transaction }) {
             decimals={decimals}
             image={tokenImage}
             tokenAddress={tokenAddress}
+            onEdit={async ({ txData }) => {
```

Comment:
Minor, it's a legacy pattern in the file but should we define this once as a `handleNftEdit` somewhere to avoid the duplication with the other paths?

Reply:
I would avoid getting into cleanup as this code is soon going to go away anyways,